bin/varnishncsa/varnishncsa.c

```diff
@@ -963,7 +963,12 @@ process_hdr(enum format_policy fp, const struct watch_head *head, const char *b,
 		CHECK_OBJ_NOTNULL(w, WATCH_MAGIC);
 		if (!isprefix(w->key, w->keylen, b, e, &p))
 			continue;
-		frag_line(fp, p, e, &w->frag);
+		if (unset) {
+			frag_line(fp, CTX.missing_string,
+			    CTX.missing_string + strlen(CTX.missing_string),
+			    &w->frag);
```

Comment:
You could pass `CTX.missing_string, NULL` and make `frag_line()` find the end of the `b` string upon seeing a null `e` argument.